CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change history for ui-search
 
-## 3.1.0 (IN PROGRESS)
+## 3.2.0 (IN PROGRESS)
+
+## [3.1.0](https://github.com/folio-org/ui-search/tree/v3.1.0) (2020-10-13)
+[Full Changelog](https://github.com/folio-org/ui-search/compare/v3.0.0...v3.1.0)
 * Refactor to `miragejs` from `bigtest/mirage`
 * Increment `@folio/stripes` to `v5.0.0`, increment `react-intl` to `v5.7.0` UISE-127.
```

Comment:
Please add a line like [this](https://github.com/folio-org/ui-search/pull/237/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR16) at the end.